src/main/resources/static/mini-manage-api-stubs.yml

```diff
@@ -134,7 +134,7 @@ paths:
         content:
           'application/json':
             schema:
-              $ref: '#/components/schemas/BookingBody'
+              $ref: '#/components/schemas/NewBooking'
```

Comment:
I think `BookingDto` might be a better name here, as this is what we're calling it in the UI

Reply:
Whoops - didn't realise this had been merged!